codetransformer/instructions.py

```diff
@@ -0,0 +1,163 @@
+from abc import ABCMeta, abstractmethod
+from dis import opname, opmap, hasjabs, hasjrel, HAVE_ARGUMENT, stack_effect
```

Comment:
If it fits on one line I think it should stay on one line; however, I agree with the other case.